src/main/java/com/courtcanva/ccfranchise/controllers/StaffController.java

```diff
@@ -0,0 +1,37 @@
+package com.courtcanva.ccfranchise.controllers;
+
+import com.courtcanva.ccfranchise.dtos.StaffVerifyEmailPostDto;
+import com.courtcanva.ccfranchise.exceptions.ExpiredVerificationTokenException;
+import com.courtcanva.ccfranchise.services.StaffService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@Slf4j
+@RestController
+@RequestMapping("/staff")
+@RequiredArgsConstructor
+public class StaffController {
+
+    private final StaffService staffService;
+
+    @PostMapping("/verification")
+    @ResponseStatus(HttpStatus.CREATED)
+    public void sendVerificationEmail(@RequestParam Long id) {
+        staffService.sendVerificationEmail(id);
+    }
+
+    @PostMapping("/verify")
```

Comment:
I suggest use put method

Reply:
It was discussed again that `POST` is probably better method for this endpoint, so no changes are made here.